### DISCUSSION TITLE
Mismatch severity

### DISCUSSION THREAD

**Discussion by imurata** [aquasecurity/trivy#2550] [Q&A] [answered]
Hi all.
I scanned my image with trivy and Harbor, but the result of vulnerability severity does not match.
Upon closer inspection, I found that the results of the trivy command was incorrect.
For example, the following result should be Medium, but the trivy shows High.

```
│ org.springframework:spring-core (spring-core-5.3.15.jar)     │ CVE-2022-22968      │ HIGH     │                   │ 5.2.21, 5.3.19         │ Spring Framework: Data Binding Rules Vulnerability         │
│                                                              │                     │          │                   │                        │ https://avd.aquasec.com/nvd/cve-2022-22968
```

Is this a bug?

DB information:
```
$ cat ~/Library/Caches/trivy/db/metadata.json
{"Version":2,"NextUpdate":"2022-07-20T12:06:37.496588142Z","UpdatedAt":"2022-07-20T06:06:37.496588442Z","DownloadedAt":"2022-07-20T07:43:57.52925Z"}
$ md5sum ~/Library/Caches/trivy/db/trivy.db
f2603109acb6268cdc446b966dc350c8  /Users/owner/Library/Caches/trivy/db/trivy.db
```

**Comment by knqyf263** [comment, marked as answer]
Please see `DataSource` in the output from `-f json`. I guess it is from GHSA.
https://github.com/advisories/GHSA-g5mm-vmx4-3rg7

**Reply by imurata** [reply]
-f json's results convinced me.
Thank you for quick reply.
BTW, I have read your article. Honored to get a reply from you. Thanks again!

**Reply by knqyf263** [reply]
Anytime.

>BTW, I have read your article. Honored to get a reply from you. Thanks again!

I'm glad to hear that😄
